railgun/manager/manager.go

```diff
@@ -75,7 +77,7 @@ func MakeFlagSetFor(c *Config) *pflag.FlagSet {
 	flagSet.StringVar(&c.FilterTag, "kong-filter-tag", "managed-by-railgun", "TODO")
 	flagSet.IntVar(&c.Concurrency, "kong-concurrency", 10, "TODO")
 	flagSet.StringVar(&c.SecretName, "secret-name", "kong-config", "TODO")
-	flagSet.StringVar(&c.SecretNamespace, "secret-namespace", controllers.DefaultNamespace, "TODO")
+	flagSet.StringVar(&c.SecretNamespace, "secret-namespace", ctrlutils.DefaultNamespace, "TODO")
```

Comment:
shall we remove these now?

Reply:
Yes indeed 4b175175e7fb52abc687de8cb81f559a052ffe10